ex5/ex5.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Write a function that given a string, returns the first character that
+ * doesn't repeat in the whole string. Otherwise (all characters repeat),
+ * should return null.
+ *
+ * For example:
+ * firstUniqueChar("abcdedcba") === "e"
+ * firstUniqueChar("abcdeedcba") === null
+ * firstUniqueChar("abababcab") === "c"
+ * firstUniqueChar("aaaaaaaaa") === null
+ * firstUniqueChar("a") === "a"
+ */
+
+function firstUniqueChar(str) {
+
+	var doNotLookForCharList = '', searchChar = '', searchIndex = 0; 
+
+	for (searchIndex in str) {
+
+		searchIndex = parseInt(searchIndex);
+
+		searchChar = str.charAt(searchIndex);
+
+		//  Let's see if we did look for searchChar
+		if (doNotLookForCharList.includes(searchChar) != true) {
```

Comment:
I am fine with the `charAt` call. I was asking you to replace the `includes`. Re-read what I wrote.